api/v1alpha3/condition_consts.go

```diff
@@ -109,9 +109,6 @@ const (
 	// MachineHasFailureReason is the reason used when a machine has either a FailureReason or a FailureMessage set on its status.
 	MachineHasFailureReason = "MachineHasFailure"
 
-	// NodeNotFoundReason is the reason used when a machine's node has previously been observed but is now gone.
-	NodeNotFoundReason = "NodeNotFound"
```

Comment:
Should we deprecate instead of deleting?

Reply:
It is moved below, not deleted.